elasticsearch/example/src/test/java/example/springdata/elasticsearch/conference/AssumeConnection.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package example.springdata.elasticsearch.conference;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * @author Prakhar Gupta
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@ExtendWith(ElasticsearchAvailable.class)
+public @interface AssumeConnection {
```

Comment:
We should follow the `EnabledOn…` naming scheme (as in `EnabledOnElasticsearchAvailable` or so).

Reply:
Already done as part of https://github.com/spring-projects/spring-data-examples/commit/b5cecf730bf8e034c2dfc8eb84695cdd331a6202